packages/cli/src/commands/source/push/delete.ts

```diff
@@ -23,8 +23,8 @@ export default class SourcePushDelete extends Command {
       char: 'd',
       exclusive: ['delete'],
       description:
-        'If this flag is set, all items that have been added or updated in the source before the specified ISO 8601 date or Unix timestamp will be deleted. The documents will be deleted using the default queueDelay, meaning they will stay in the index for about 15 minutes after being marked for deletion.',
-      helpValue: '2000-01-01T00:00:00-06:00 OR 946702800',
+        'If this flag is set, all items that have been added or updated in the source before the specified ISO 8601 date or timestamp in milliseconds will be deleted. The documents will be deleted using the default queueDelay, meaning they will stay in the index for about 15 minutes after being marked for deletion.',
```

Comment:
Unsure about that, what motivates this change?

Reply:
The `orderingId` should be a timestamp in milliseconds (see [swagger docs](https://platformdev.cloud.coveo.com/docs?urls.primaryName=PushAPI#/Item/delete_organizations__organizationId__sources__sourceId__documents_olderthan)). Otherwise it will not work.
@francistb can you confirm?

Reply:
orderingId can be customer provided, it can be any number. 

our crawlers, and pushapi if you don't specify anything, use a timestamp in milliseconds (something like 1638541545341)

Reply:
![image](https://user-images.githubusercontent.com/757840/144631331-bb2bd7f5-6831-4c55-87a6-baf5b7211745.png)
ratlab in dev for example of custom ordering

Reply:
```suggestion
        'If this flag is set, all items that have been added or updated in the source before the specified ISO 8601 date or Unix timestamp in milliseconds will be deleted. The documents will be deleted using the default queueDelay, meaning they will stay in the index for about 15 minutes after being marked for deletion.',
```

Specify that it's a Unix timestamp. I know and agree that it's quite obvious and that Unix/Epoch is the ubiquitous one, but there's other reference point out-there, so let's be precise when we can without much fluff (which is the case here)